fast64_internal/f3d/f3d_parser.py

```diff
@@ -2223,6 +2222,9 @@ def parseMacroArgs(data: str):
             params.append(param)
             start = end + 1
 
+    if len(data) == 1:
+        return [data]
```

Comment:
parseMacroArgs when he have in input "1" he return [] like there is any argument because he didn't enter in the while loop, but I am agree it's not the best way to fix the problem

Reply:
I think I miss something on how the loop are made I will just remove it